portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,13 @@
+package com.google.sps.data;
+
+public class Comment {
+    private String username;
+    private String commentText;
+    private boolean isAdmin;
+
+    public Comment (String username, String commentText, boolean isAdmin) {
+        this.username = username;
+        this.commentText = commentText;
+        this.isAdmin = isAdmin;
```

Comment:
This value isn't being used, so just remove it.

Reply:
Removed